### DISCUSSION TITLE
Feature Request: ONE ROW PER VERTEX

### DISCUSSION THREAD

**Discussion by Dtenwolde** [duckdb/duckdb#7757] [Ideas]
Why do you want this feature?
alternative to default ONE ROW PER MATCH semantics, SQL/PGQ also adopted the ONE ROW PER VERTEX or ONE ROW PER STEP from Oracle’s PGX (https://pgql-lang.org/spec/2.0/) to avoid dealing with lists for paths

**Comment by hannes** [comment]
Hey, thanks for filing this, we will consider this if we should decide to add SQL/PGQ into mainline DuckDB.
